CODEGEN.md

```diff
@@ -86,7 +86,7 @@ gen.writeFiles("src/main/scala/com/my/project")
 You can parse column and table names using either the `SnakeCaseNames` or the and the `LiteralNames` parser
 which are used with the respective Quill Naming Strategies. They cannot be customized further with this code generator.
 
-The following case case classes will be generated
+The following case classes will be generated
```

Comment:
Sometimes doubled words are intended. In this repository, the only instance I found that seemed to need it was `sbt sbt` where the first was a docker-compose service and the second was a command.